frontend/src/components/compiler/compiler_settings/gcc.tsx

```diff
@@ -0,0 +1,15 @@
+import { Checkbox, FlagSet, FlagOption } from "../CompilerOpts"
```

Comment:
It's good to have a common gcc settings file like this, and we should probably use it in all the other gcc compilers too